portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>sehhaj portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>favourite things</h1>
+      <div id ="gallery">
+        <img src="./images/pd.jpg" alt="me when i drive">
+      </div>
+      <div id ="gallery">
+        <img src="./images/art.jpeg" alt="art">
+      </div>
+      <div id ="gallery">
+        <img src="./images/art2.jpeg" alt="swirly art">
+      </div>
+      <div id ="gallery">
+        <img src="./images/dog.jpeg" alt="doggo">
```

Comment:
When you hover over these images because you're increasing the boarder it's changing the layout of the images, particularly when you hover over this dog one. I suspect this is not intentional, you might be able to make it work with a non-block layout, but I think the better option would be to just keep the border size fixed and just change it's colour.

Reply:
Played around with css styling of the gallery. Ended up rewriting the code because I wasn't happy with how it looked.